spread.yaml

```diff
@@ -263,6 +263,11 @@ suites:
    systems:
      - ubuntu-24.04*
 
+ tests/spread/core24-suites/plugins/:
+   summary: core24 plugin tests
+   systems:
+     - ubuntu-24.04*
```

Comment:
do these tests build destructively?
One argument "pro" using something lower than 24.04 is that it lets us catch failures in classic snaps